src/scenes/home/landing/triGroup/triGroup.css

```diff
@@ -0,0 +1,51 @@
+.videos {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  border-radius: 4px;
+  overflow: hidden;
+}
+
+.triGroupContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  background-color: #f0f2f2;
+  padding-bottom: 30px;
+}
+
+.clickableLogo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+iframe {
```

Comment:
We can't apply styles to elements. Please apply these styles via creating a CSS class. As it currently stands, those inserted styles will affect every iFrame on the site.